src/backend/marsha/core/api.py

```diff
@@ -587,10 +589,17 @@ def update_live_state(
             live_info.update({"stopped_at": stamp})
             video.upload_state = defaults.HARVESTING
             video.live_info = live_info
-            create_mediapackage_harvest_job(video)
             delete_aws_element_stack(video)
             if settings.LIVE_CHAT_ENABLED:
                 close_room(video.id)
+            try:
+                create_mediapackage_harvest_job(video)
+            except ManifestMissingException:
+                delete_mediapackage_channel(video)
+                video.upload_state = defaults.DELETED
+                video.live_state = None
```

Comment:
Should there be a state for this? Should the state ever be None after creation? Looks like a lie :wink:

Reply:
the `DELETED` upload state is here for that I think.